akka-docs/rst/scala/code/docs/http/scaladsl/HttpServerExampleSpec.scala

```diff
@@ -380,4 +381,36 @@ class HttpServerExampleSpec extends WordSpec with Matchers {
       }
     }
   }
+
+  "all-positive-numbers" in compileOnlySpec {
+    //#all-positive-numbers
+    import akka.stream.scaladsl._
+    import akka.util.ByteString
+    import akka.http.scaladsl.Http
+    import akka.http.scaladsl.model.HttpEntity
+    import akka.http.scaladsl.server.Directives._
+    import akka.stream.ActorMaterializer
+
+    implicit val system = ActorSystem()
+    implicit val materializer = ActorMaterializer()
+
+    // streams are re-usable so we can define it here and use it for every request
+    val numbers = Source.fromIterator(() => Iterator.iterate(1)(n => n + 1))
+
+    val route =
+      path("numbers") {
+        get {
+          complete {
```

Comment:
Usually we have `complete()` (with `()`) as it's kind of special (so it doesn't get lost among "just some directives")